.github/workflows/lint.yml

```diff
@@ -47,6 +47,8 @@ jobs:
       - uses: actions-rust-lang/setup-rust-toolchain@v1
 
       - uses: EmbarkStudios/cargo-deny-action@v1
+        with:
+          rust-version: "1.84.0"
```

Comment:
@JamyGolden Looks like `cargo-deny-action` uses an older Rust version than what `setup-rust-toolchain` uses for the rest.